src/web/mjs/connectors/Qiman5.mjs

```diff
@@ -0,0 +1,74 @@
+import SinMH from './templates/SinMH.mjs';
+
+export default class Qiman5 extends SinMH {
+
+    constructor() {
+        super();
+        super.id = 'qiman5';
+        super.label = '奇漫屋 (Qiman5)';
+        this.tags = [ 'webtoon', 'chinese'];
+        this.url = 'http://www.qiman6.com';
+
+        this.path = '/rank/1-%PAGE%.html';
+        this.queryMangas = 'div.mainForm div.updateList div.bookList_3 div.ib p.title a';
+        this.queryChapters = 'div.chapterList div#chapter-list1 a.ib';
+        this.queryManga = 'div.container div.mainForm div.comicInfo div.ib.info h1';
+        this.queryChaptersScript = `
+            new Promise((resolve, reject) => {
+                setTimeout(() => {
```

Comment:
May use sequential timeouts instead of nested timeouts
```javascript
setTimeout(() => { /* Click Button */ }, 1000);
setTimeout(() => { /* Extract Chapters from DOM */ }, 5000);
```

Reply:
Mh... the change doesn't make them sequential, they are still nested but in the end it doesn't impact the functionality